ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml

```diff
@@ -64,6 +65,7 @@ spec:
         if [ -f "test_results.json" ]; then
           echo "Results exists"
           echo -n "true" | tee $(results.preflight_results_exists.path)
+          cat test_results.json | tee $(results.preflight_results.path)
```

Comment:
It is better to return just a boolean flag with a result instead of a whole json.

Reply:
We need this json for verify-ci-results task.